fms/bin/main.py

```diff
@@ -55,42 +55,45 @@ def _build_commands():
 
 def main():
     top_parser = argparse.ArgumentParser(
```

Comment:
Question: it looks like we have two sets of cli entrypoints in this repo. Which one is the one that gets used?

Reply:
deleted unused main.py